Superposition/ReferenceImplementation.qs

```diff
@@ -115,45 +115,55 @@ namespace Quantum.Kata.Superposition {
             X(qs[1]);
         }
     }
-
-
+    
+    
     // Task 8. Greenberger–Horne–Zeilinger state
     // Input: N qubits in |0...0⟩ state.
     // Goal: create a GHZ state (|0...0⟩ + |1...1⟩) / sqrt(2) on these qubits.
     operation GHZ_State_Reference (qs : Qubit[]) : Unit is Adj {
-
+        
         H(qs[0]);
-
+            
         for (q in Rest(qs)) {
             CNOT(qs[0], q);
         }
     }
-
-
+    
+    
     // Task 9. Superposition of all basis vectors
     // Input: N qubits in |0...0⟩ state.
     // Goal: create an equal superposition of all basis vectors from |0...0⟩ to |1...1⟩
     // (i.e. state (|0...0⟩ + ... + |1...1⟩) / sqrt(2^N) ).
     operation AllBasisVectorsSuperposition_Reference (qs : Qubit[]) : Unit is Adj {
-
+        
         for (q in qs) {
             H(q);
         }
     }
 
-
+	// Task 10. Superposition of all even numbers
```

Comment:
Please use 4 spaces instead of tabs for indentation to be consistent with the rest of the file. (The same applies to the changes in `Tasks.qs` and `Tests.qs`)